File: src/java/eu/sig/training/ch06/digitalcamera/DigitalCamera.java
package eu.sig.training.ch06.digitalcamera;

import java.awt.Image;
import java.awt.image.BufferedImage;

// tag::DigitalCamera[]
public class DigitalCamera {
    public Image takeSnapshot() {
        // ...
        // end::DigitalCamera[]
        return new BufferedImage(1920, 1080, BufferedImage.TYPE_INT_ARGB);
        // tag::DigitalCamera[]
    }

    public void flashLightOn() {
        // ... 
    }

    public void flaslLightOff() {
        // ... 
    }
}
// end::DigitalCamera[]

File: src/csharp/eu/sig/training/ch06/digitalcamera/DigitalCamera.cs
using System.Drawing;

namespace eu.sig.training.ch06.digitalcamera
{

    // tag::DigitalCamera[]
    public class DigitalCamera
    {
        public Image TakeSnapshot()
        {
            // ...
            // end::DigitalCamera[]
            return Image.FromFile("");
            // tag::DigitalCamera[]
        }

        public void FlashLightOn()
        {
            // ...
        }

        public void FlashLightOff()
        {
            // ...
        }
    }
    // end::DigitalCamera[]
}
